src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx

```diff
@@ -59,21 +59,35 @@ To install the Node.js agent:
 
 1. Create a New Relic account. Don't have one? [Sign up for free!](https://newrelic.com/signup) No credit card required.
 2. Ensure you meet the [system requirements](/docs/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent). In particular, make sure you use a supported Node.js version.
-3. Use the command `npm install newrelic --save` for each application you want to monitor.
+3. Use the command `npm install newrelic` for each application you want to monitor.
+  <Callout variant="important">
+    If you're using Next.js use our standalone [`@newrelic/next` agent](https://github.com/newrelic/newrelic-node-nextjs) instead of the `newrelic` agent. [Here is an example Next.js app](https://github.com/newrelic-experimental/newrelic-nextjs-integration)
```

Comment:
Isn't this the repo that's tied to the misleading blog post? (Though apparently we still link to that example in our ReadMe.)

Reply:
@bizob2828 sent me a link to that one as an example of the Next instrumentation, but it is made by Mat as well. The Nest instrumentation was the one we just removed. 

Honestly I mix up Nest and Next constantly so I don't recall if this repo has issues.  If there's any doubts about that repo I can remove it, but it is currently linked as an example in the official next agent readme:

- https://github.com/newrelic/newrelic-node-nextjs#example-project